CONTRIBUTING.md

```diff
@@ -10,15 +10,25 @@ Welcome! We are very happy to accept community contributions to Ratify, whether
 * Checkout the repo locally with `git clone git@github.com:{your_username}/ratify.git`.
 * Build the Ratify CLI with `go build -o ./bin/ratify ./cmd/ratify` or if on Mac/Linux/WSL `make build-cli`.
 
+
+## Feature Enhancements
+For non-trivial enhancements or bug fixes, please start by raising a design PR. You can refer to the example [here](https://github.com/ratify-project/ratify/blame/dev/docs/proposals/Release-Supply-Chain-Metadata.md).
```

Comment:
wonder if we should add design doc under  docs/design or docs/proposal path?

Reply:
Hi @binbin-li , I want to get a common understanding on the difference between /design and proposal. I m thinking proposal doc involving customer experience, design for technical design. Community contributors likely to author one doc for simplicity. ( we should not over burden contributors with extra process). Should we guide them to /design or /proposal